h2o-docs/src/product/blog-releases.rst

```diff
@@ -577,3 +577,88 @@ Prior Release Blogs
 ~~~~~~~~~~~~~~~~~~~
 
 You can find all prior release blogs `here <https://h2o.ai/blog/category/h2o-release/>`__.
+
+General Blogs
+-------------
+
+A Look at the UniformRobust method for ``histogram_type``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We introduced the `UniformRobust method for histogram <data-science/algo-params/histogram_type.html>`__ (``histogram_type="UniformRobust"``) in 3.36.1. and want to highlight its benefits! This method addresses issues found in the default UniformAdaptive method and preserve the runtime performance advantages compared to the QuantilesGlobal method.
+
+The issue found with the default UniformAdaptive method deals with outlier issues related to binning. This issue can be seen in how the UniformAdaptive method handles certain datasets with many columns that are primarily comprised of small integers where there are also columns with artificial values upwards of 99999, 99998. While you can certainly handle this issue in pre-processing steps, the default binning method can fail on features with certain characteristics. Uniform binning looks at the min and max values in the column from the observations in the current tree node. It splits the interval into bins of the same length. However, when there are such large outliers, these outliers are classified into a single bin while the rest of the observations end up in another single bin. This, unfortunately, leaves most of the bins unused.
+
+The introduction of the UniformRobust method mitigates these issues! By learning from histograms from the previous layer, we are able to fine-tune the split points for the current layer.
+
+To begin, the root of the tree uses UniformAdaptive binning. After the first split is made, we learn how the histograms were populated. Though this split has been made, the data has not yet been seen, so we do not yet know the exact distributions of the data in the right and left child nodes. If we see that UniformAdaptive binning fails, we can assume an issue still exists in the child nodes. Then, we approximate the distribution in the child nodes by the distribution in the parent node. We do this by looking at the bins of the histogram to check how many bins were empty compared to the total number of available bins. If too few bins are populated, that is when we switch to the new binning method.
```

Comment:
What does it mean for Uniform Adaptive binning to fail?

Reply:
My guess is along the lines of "fails to distribute/bin evenly"? But I am just going off what is available in the [original PR](https://github.com/h2oai/h2o-3/pull/6063). 

@maurever any guidance?

Reply:
I wasn't sure how in-depth to go with this explanation, so I did start with the minimum:
`If we see that UniformAdaptive binning fails (i.e. there are a lot of empty bins)...` 
but please let me know if I should add more.